jenkins/java/ql/src/HasPermissionReturnIgnored.md

```diff
@@ -1,7 +1,25 @@
-== Ignoring return value of `#hasPermission` calls
-
 This code calls `#hasPermission` without checking the return value. This call should either be `#checkPermission`, or have its return value used.
 
+## Why is this a problem?
+
+This is indicative of a bug: Code intended to ensure the user accessing some functionality has a given permission, but ignoring the return value of this method means that anyone would be able to proceed.
```

Comment:
I would suggest shorter phrases. How about: 

"This is indicative of a bug. The code  wants to ensure that, to access a functionality, the user  holds a given permission. But the method's return value is ignored. This means that anyone can access the functionality."